webpack/externals.js

```diff
@@ -4,6 +4,17 @@
  * @since 2.0.0
  */
 
+/**
+ * Convert any string to camelcase.
+ *
+ * @param string str String to convert
+ *
+ * @since 2.0.0
+ */
+function camalize(str) {
```

Comment:
Is it intentional that is Camalize and not Camelize ?

Reply:
fixed